.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,5 +1,5 @@
 connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
-connection.project.dir=
+connection.project.dir=../xtext-umbrella.git/releng/gradle-composite
```

Comment:
Would it also work on my machine? I do not have this repository in parent folder.